libcloud/common/google.py

```diff
@@ -111,6 +116,15 @@ def _is_gce():
     return False
 
 
+def _is_gcs_s3(user_id):
+    """Checks S3 key format: 20 alphanumeric chars starting with GOOG."""
+    return bool(re.match(r'GOOG[A-Z0-9]{16}', user_id))
```

Comment:
nit: probably good enough to just use len() and user_id.startswith vs pulling in the `re` module